src/css/style.css

```diff
@@ -471,6 +470,22 @@ button {
     content: "\e60c";
 }
 
+.gpii-fd-dummy:before {
```

Comment:
Thanks for addressing all comments quickly, @amb26. The css for `gpii-fd-dummy` can be removed too with the removal of the dummy icon.